internal/operator/namespace_test.go

```diff
@@ -0,0 +1,37 @@
+package operator
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Namespace", func() {
+	var oc Client
+	var err error
+
+	BeforeEach(func() {
+		oc, err = NewOpCapClient()
```

Comment:
you should probably be using a mock opcap client here 🤔

Reply:
There is an issue already open for creating mock clients: https://github.com/opdev/opcap/issues/74